roles/openshift_health_checker/test/elasticsearch_test.py

```diff
@@ -50,10 +54,10 @@ def assert_error(error, expect_error):
 
 
 def test_check_elasticsearch():
-    assert 'No logging Elasticsearch pods' in canned_elasticsearch().check_elasticsearch([])
+    assert 'No logging Elasticsearch pods' in elasticsearch().check_elasticsearch([])
```

Comment:
> But since it seems the intention is to call the function with different args, then perhaps it doesn't even need to be a fixture, but simply a plain function?

Agree, will update